src/app/components/Home/carousel.tsx

```diff
@@ -0,0 +1,44 @@
+'use client'
+import { CarouselLists } from '@/app/constants'
+import Image from 'next/image'
+import Carousel from 'react-bootstrap/Carousel'
+
+export function HomepageCarousel() {
+  return (
+    <div>
+      <Carousel fade>
+        {CarouselLists.map(function (item) {
+          return (
```

Comment:
Use arrow function please. Do not forget `key` prop.

Reply:
done